### ISSUE TITLE
Create JSX component for displaying store items

### ISSUE THREAD

**Issue by theawesomew** [coopsoc/website#17]
Probably just a card element to display items which are available. We have similar 'matrices' of images/content in other places and we can kind of steal the idea from there too (see the team.jsx page and the card we display there?)

**Comment by scorpiontornado** [comment]
Could we also add things like sort by price, filter by colour etc?

**Comment by theawesomew** [comment]
Yeah, I think that would be a really good enhancement to the website but, we won't have an abundance of products that would need filtering. But, if we get it done and we have extra time, I'm absolutely down for adding that feature. If we do that then [this issue](https://github.com/coopsoc/coopsoc.com.au/issues/16) will be a 1000% yes.
